sklearn/metrics/scorer.py

```diff
@@ -33,14 +34,15 @@
 
 
 class _BaseScorer(six.with_metaclass(ABCMeta, object)):
-    def __init__(self, score_func, sign, kwargs):
+    def __init__(self, score_func, sign, kwargs, deprecation_msg=None):
```

Comment:
Maybe we could rename that parameter to `_deprecation_msg` to make it explicit that this is not public API.